js/main.js

```diff
@@ -0,0 +1,18 @@
+const menuBars = document.getElementsByClassName('menu-bars')[0];
+const mobileMenu = document.getElementsByClassName('mobile-menu')[0];
+const header = document.getElementById('header');
+const closeBtn = document.getElementById('hamBtn');
+
+menuBars.addEventListener('click', () => {
+  if (mobileMenu.classList.contains('open')) {
+    mobileMenu.style.display = 'none';
+    closeBtn.className = 'fas fa-bars';
+    mobileMenu.classList.remove('open');
+    header.classList.remove('active');
+  } else {
+    mobileMenu.style.display = 'flex';
+
+    mobileMenu.classList.add('open');
+    header.classList.add('active');
+  }
```

Comment:
- [ ] In the requirements, when the user clicks any of the mobile menu options, the mobile menu should disappear, but here, it only disappears once you click on the menu button again.